### DISCUSSION TITLE
Missing Container and Service Error

### DISCUSSION THREAD

**Discussion by jtrodriguez** [Security-Onion-Solutions/securityonion#4180] [Unsupported Versions]
Version 2.3.5
CentOS 7
Manager Node

so-elastalert status "ERROR"
so-suricata status "MISSING"

I have run so-docker-refresh and rebooted the machine with no luck. Does anyone have any suggestions? I have posted some of the error information below.

I have attempted so-docker-refresh and sudo docker stop so-dockerregistry / sudo docker rm so-dockerregistry /sudo salt-call state.highstate

![image](https://user-images.githubusercontent.com/73293299/118025672-19f0a180-b32e-11eb-85b9-6f2c20f19615.png)

salt-call state.highstate : 1 Failure

  
ID: prune_images
Function: cmd.run
Name: so-docker-prune
Result: False
Comment: Command "so-docker-prune" run
Started: 15:33:31.192934
Duration: 448.459 ms
Changes:

sudo so-suricata-start
Starting suricata...

This could take a while if another Salt job is running.
Run this command with --force to stop all Salt jobs before proceeding.
[INFO    ] Loading fresh modules for state activity
[INFO    ] Running state [suricata_state_not_allowed] at time 15:35:18.463223
[INFO    ] Executing state test.fail_without_changes for [suricata_state_not_allowed]
[ERROR   ] Failure!
[INFO    ] Completed state [suricata_state_not_allowed] at time 15:35:18.464586 (duration_in_ms=1.365)
local:

          ID: suricata_state_not_allowed
    Function: test.fail_without_changes
      Result: False
     Comment: Failure!
     Started: 15:35:18.463221
    Duration: 1.365 ms
     Changes:

Summary for local

Succeeded: 0
Failed:    1

Total states run:     1
Total run time:   1.365 ms

sudo docker logs so-suricata
Error: No such container: so-suricata

**Comment by jtrodriguez** [comment]
It appears that one of my alerts was causing the so-elastalert to ERROR out. I don't see how since I had not changed any recently. I am going to add them back into the alerts folder one by one to see which one is the culprit.

The so-suricata is still MISSING though.

**Reply by cm-ops** [reply]
Suricata does not run on a Manager node, that is why the state is failing. You could try the following:
`sudo docker stop so-suricata`
`sudo docker rm so-suricata`

Edit `/opt/so/conf/so-status/so-status.conf` and remove the so-suricata line. Rerun so-status and verify.

**Reply by jtrodriguez** [reply]
Do you know if this was a recent update that removed it from the manager node? It seems odd that it was considering "OK" for the longest time then it went missing. The docker image is still there but the container is gone

sudo docker rm so-suricata
Error: No such container: so-suricata

I did remove it from the configuration file to hide the error though. Thanks for that tip!
